packages/backend/src/graphql/resolvers/auto-peering.test.ts

```diff
@@ -20,6 +19,8 @@ import { CreateOrUpdatePeerByUrlInput } from '../generated/graphql'
 import { AutoPeeringService } from '../../payment-method/ilp/auto-peering/service'
 import { v4 as uuid } from 'uuid'
 
+const nock = (global as unknown as { nock: typeof import('nock') }).nock
```

Comment:
What about this?

add this to  a new file at `rafiki/packages/backend/src/global.d.ts`:

```ts
declare module globalThis {
  var nock: typeof import('nock')
}
```

which lets us just do:


```ts
const nock = global.nock
```

Reply:
Although I dont love that this suggests nock is on global *anywhere* (including outside tests, or outside the that specific test environment) ... was just thinking maybe we could be easier to reference than doing all this casting everywhere. alternatively maybe we can initialize it like this once somewhere and export it?

Reply:
I'm not too fond of this long casting, but I think I prefer not mixing the shipped code vs the testing code IMO